cql3/statements/cf_prop_defs.cc

```diff
@@ -106,7 +108,8 @@ void cf_prop_defs::validate(const database& db, const schema::extensions_map& sc
         KW_GCGRACESECONDS, KW_CACHING, KW_DEFAULT_TIME_TO_LIVE,
         KW_MIN_INDEX_INTERVAL, KW_MAX_INDEX_INTERVAL, KW_SPECULATIVE_RETRY,
         KW_BF_FP_CHANCE, KW_MEMTABLE_FLUSH_PERIOD, KW_COMPACTION,
-        KW_COMPRESSION, KW_CRC_CHECK_CHANCE, KW_ID, KW_PAXOSGRACESECONDS
+        KW_COMPRESSION, KW_CRC_CHECK_CHANCE, KW_ID, KW_PAXOSGRACESECONDS,
+        KW_TOMBSTONE_GC
     });
```

Comment:
We need to block this on a mixed-version cluster. It's unlikely that someone will run the command on a mixed cluster, but the results are likely a total crash, so need to protect against it.

Reply:
Block at which point? Is blocking  ALTER TABLE and CREATE table with the KW_TOMBSTONE_GC opiton good enough?

Reply:
Yes

Reply:
So, we introduce a gossip feature TOMBSTONE_GC. Check if this gosisp feature is enabled to decide if we can use KW_TOMBSTONE_GC.  Looks OK? 

What did we do for other existing options (any thing we added which is not from C*)?

Reply:
Generally that's what we did for anything that requires cross-node compatibility. CQL features which are fully implemented in the coordinator don't need it.

We should replace the gossip feature with something lighter weight for these use cases, one day.